deepquest/train.py

```diff
@@ -14,7 +14,7 @@
 from nmt_keras.nmt_keras import check_params
 
 import deepquest.qe_models as modFactory
-from deepquest.utils.callbacks import PrintPerformanceMetricOnEpochEndOrEachNUpdates
+from deepquest.utils.callbacks import PrintPerformanceMetricOnEpochEndOrEachNUpdates#, EarlyStopping
```

Comment:
that seems untidy

Reply:
Thanks for the spot, this is left over from some testing. Will go through and clean it up before merging.